Cargo.toml

```diff
@@ -40,6 +40,10 @@ features = ["xlib"]
 version = "2.3.0"
 features = ["xlib"]
 
+[target.armv7-unknown-linux-gnueabihf.dependencies.x11]
```

Comment:
It's legal to do
```
[target.cfg(target_os = "linux").dependencies]
x11 = { version = "2.3.0", features = ["xlib"] }
```
and that would allow us to remove all of these duplicate entries, I'm pretty sure.